src/xrpl-cheat-sheet.json

```diff
@@ -370,6 +370,22 @@
                 "description": ""
             }
         ],
+        "Price Oracles": [
+            {
+                "name": "XRPL Labs Price Aggregator",
+                "link": "https://github.com/XRPL-Labs/XRPL-Persist-Price-Oracle",
+                "description": ""
+            },
+            {
+                "name": "XRP-Oracles",
+                "link": "https://github.com/lathanbritz/XRP-Oracles"
```

Comment:
Add description field here as well

Reply:
Also it'll be great if you add a short description :)

Reply:
I created "Price Oracles" under "Dev Utilities". Subcategories don't seem to support a description field (no other example).
All three project basically do the same. A description would be identical for all.

Reply:
merging it, I will suggest paraphrasing